syntax = "proto3";

package mails;

message FullStorageIssue {
  User user = 1;
  string disk_allowed = 2;
}
